ingest-app/src/cmr/ingest/services/messages.clj

```diff
@@ -49,6 +49,15 @@
    (format "Data center short name [%s] was not a valid keyword."
           (:ShortName data-center)))
 
+(defn data-format-not-matches-kms-keywords
+  "Error message when FileDistributionInformation -> Format is
+  not in the KMS."
+  [archive-and-distribution-information]
+  (format "Format [%s] within ArchiveAndDistributionInformation [%s] was not a valid keyword."
```

Comment:
I don't think it is a good error message. The ArchiveAndDistributionInformation might be too much for an error message.

Reply:
Can we add a test for validating granule data format with KMS values? Then we can see what this error message really looks like.

Reply:
Fixed and added tests